services/tlsinfo/client/client.go

```diff
@@ -103,20 +106,29 @@ func (c *getCertsCmd) Execute(ctx context.Context, f *flag.FlagSet, args ...inte
 
 	foundErr := false
 	for r := range respChan {
-		fmt.Fprintf(state.Out[r.Index], "Target %s result:\n", r.Target)
 		if r.Error != nil {
 			fmt.Fprintf(state.Err[r.Index], "Get TLS certificates failure: %v\n", r.Error)
 			foundErr = true
 			continue
 		}
 		for i, cert := range r.Resp.Certificates {
-			fmt.Fprintf(state.Out[r.Index], "---Server Certificate--- %d\n", i)
-			fmt.Fprintf(state.Out[r.Index], "Issuer: %v\n", cert.Issuer)
-			fmt.Fprintf(state.Out[r.Index], "Subject: %v\n", cert.Subject)
-			fmt.Fprintf(state.Out[r.Index], "NotBefore: %s\n", time.Unix(cert.NotBefore, 0))
-			fmt.Fprintf(state.Out[r.Index], "NotAfter: %s\n", time.Unix(cert.NotAfter, 0))
-			fmt.Fprintf(state.Out[r.Index], "DNS Names: %v\n", cert.DnsNames)
-			fmt.Fprintf(state.Out[r.Index], "IP Addresses: %v\n\n", cert.IpAddresses)
+			if c.printPEM {
+				err := pem.Encode(state.Out[r.Index], &pem.Block{
+					Type:  "CERTIFICATE",
+					Bytes: cert.Raw,
+				})
+				if err != nil {
+					fmt.Fprintf(state.Err[r.Index], "unable to encode cert as pem: %v\n", cert.Raw)
+				}
+			} else {
+				fmt.Fprintf(state.Out[r.Index], "---Server Certificate--- %d\n", i)
+				fmt.Fprintf(state.Out[r.Index], "Issuer: %v\n", cert.Issuer)
+				fmt.Fprintf(state.Out[r.Index], "Subject: %v\n", cert.Subject)
+				fmt.Fprintf(state.Out[r.Index], "NotBefore: %s\n", time.Unix(cert.NotBefore, 0))
+				fmt.Fprintf(state.Out[r.Index], "NotAfter: %s\n", time.Unix(cert.NotAfter, 0))
+				fmt.Fprintf(state.Out[r.Index], "DNS Names: %v\n", cert.DnsNames)
+				fmt.Fprintf(state.Out[r.Index], "IP Addresses: %v\n\n", cert.IpAddresses)
```

Comment:
What do you thing about styling? My guess it would be much easier to read if each line title would be in different color from value

Reply:
Sure, let's colorize! I've updated to use your new helper